main.go

```diff
@@ -62,6 +63,7 @@ func main() {
 	flag.StringVar(&container, "container", "", "container id for the shim")
 	flag.UintVar(&pid, "pid", 0, "process id for the shim")
 	flag.BoolVar(&proxyExitCode, "proxy-exit-code", true, "proxy exit code of the process")
+	flag.BoolVar(&proxyWinsize, "proxy-winsize", false, "proxy winsize of the tty")
```

Comment:
This name isn't very clear as it suggests this options allows the size of the window to be specified (but it clearly doesn't as it's a `bool` ;)